calico_cni/calico_cni.py

```diff
@@ -67,7 +69,14 @@ def create(args):
     subnet = args['subnet']
 
     _log.info('Configuring pod %s' % container_id)
-    netns_path = '%s/%s/%s' % (NETNS_ROOT, container_id, netns)
+
+    if CNI_NETNS_ROOT:
```

Comment:
I think you could eliminate this if/else if you set the default value of CNI_NETNS_ROOT to be RKT_NETNS_ROOT.